framework/db/conditions/InConditionBuilder.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace yii\db\conditions;
+
+use yii\db\ExpressionBuilderInterface;
+use yii\db\ExpressionBuilderTrait;
+use yii\db\ExpressionInterface;
+use yii\db\Query;
+
+/**
+ * Class InConditionBuilder builds objects of [[InCondition]]
+ *
+ * @author Dmytro Naumenko <d.naumenko.a@gmail.com>
+ * @since 2.0.14
+ */
+class InConditionBuilder implements ExpressionBuilderInterface
+{
+    use ExpressionBuilderTrait;
+
+    /**
+     * Method builds the raw SQL from the $expression that will not be additionally
+     * escaped or quoted.
+     *
+     * @param ExpressionInterface|InCondition $condition the expression to be built.
+     * @param array $params the binding parameters.
+     * @return string the raw SQL that will not be additionally escaped or quoted.
+     */
+    public function build(ExpressionInterface $condition, &$params = [])
+    {
+        $operator = $condition->getOperator();
+        $column = $condition->getColumn();
+        $values = $condition->getValues();
+
+        if ($column === []) {
+            // no columns to test against
+            return $operator === 'IN' ? '0=1' : '';
+        }
+
+        if ($values instanceof Query) {
+            return $this->buildSubqueryInCondition($operator, $column, $values, $params);
+        }
+
+        if (!is_array($values) && !$values instanceof \Traversable) {
+            // ensure values is an array
+            $values = (array) $values;
+        }
+
+        if ($column instanceof \Traversable || ((is_array($column) || $column instanceof \Countable) && count($column) > 1)) {
+            return $this->buildCompositeInCondition($operator, $column, $values, $params);
+        } elseif (is_array($column)) {
```

Comment:
No need for else since previously there was a return.

Reply:
It will be added during release process automatically